storage/diskmetricstore.go

```diff
@@ -61,6 +62,10 @@ type mfStat struct {
 	copied bool // Has the MetricFamily already been copied?
 }
 
+// legacyGroupingKeyToMetricGroup is like GroupingKeyToMetricGroup but uses the
+// old uint64 grouping key (a hash value) instead of a string grouping key.
+type legacyGroupingKeyToMetricGroup map[uint64]MetricGroup
```

Comment:
So you keep `legacyGroupingKeyToMetricGroup` type in this file instead of in `storage/interface.go` together `GroupingKeyToMetricGroup` type to make it easier to remove it (because is all together) later on? what's the reasoning behind?

Reply:
I don't want to export it. It's just a private type to handle the conversion. An unexported type in a file called `interface.go` with otherwise all the exported types would be weird.